src/backend/distributed/citus--8.3-1--8.4-1.sql

```diff
@@ -209,13 +214,20 @@ BEGIN
         RETURNING
             type,
             object_names,
-            object_args
+            object_args,
+			distribution_arg_index,
+			colocation_id
     )
-    INSERT INTO citus.pg_dist_object (classid, objid, objsubid)
+    INSERT INTO citus.pg_dist_object (
+		classid, objid, objsubid,
+		distribution_arg_index,
+		colocation_id)
     SELECT
         address.classid,
         address.objid,
-        address.objsubid
+        address.objsubid,
+		address.distribution_arg_index,
+		address.colocation_id
```

Comment:
We should probably move this statement into its own function in case we need to change it again later.